### ISSUE TITLE
extremely slow start of hashing (compared to xmr-stak)

### ISSUE THREAD

**Issue by aleqx** [xmrig/xmrig-nvidia#154]
I'm using Ubuntu x64. I have 13x GTX1070 on a motherboard. They are all overclocked, but nothing too pretentious (+100, +1200).

xmr-stak loads the GPU memory and starts hashing immediately, I can see the hashrate by pressing "H" after 2-3 seconds.

xmrig takes absolutely ages - several minutes - before hashing starts on all cards.

Watching the process using nvidia-smi, I can see that xmrig takes about 10 seconds to load a single card out of 13, then another 10-20 seconds to load the second one, and so on, sometimes loading more cards at the same time. In total, it needs several minutes before mining starts on all cards.

Something is definitely wrong here given that xmr-stak has no issues.

I also tried various values for `--cuda-bsleep=N` but nothing helped.

**Comment by aleqx** [comment]
Anyone?

**Comment by cyvitor** [comment]
have u tried the default settings before customize? and the default settings of xmrig?

**Comment by aleqx** [comment]
Yes. That's what I meant by "I also tried ..." (i.e. I also tried to custom settings). There might be issues with systems having that many GPUs. Anyone else with 13 or so GPUs on xmrig?

As I said, xmr-stak has no such issues.

**Comment by aleqx** [comment]
@xmrig any clues?

**Comment by aleqx** [comment]
Does anyone have any ideas? @xmrig? Anyone?

**Comment by aleqx** [comment]
@xmrig bump?
